projects/packages/jetpack-mu-wpcom/src/features/wpcom-admin-menu/wpcom-admin-menu.php

```diff
@@ -472,3 +473,26 @@ function_exists( 'wpcom_site_has_feature' ) &&
 	}
 }
 add_action( 'admin_menu', 'wpcom_add_plugins_menu' );
+
+/**
+ * Adds a submenu item for Jetpack Forms.
+ */
+function add_submenu_jetpack_forms() {
+	$has_switch_class = class_exists( 'Dashboard_View_Switch' );
+
+	if ( ! $has_switch_class || ! Dashboard_View_Switch::is_jetpack_forms_admin_page_available() ) {
+		return;
+	}
+
+	// This is so we don't just hardcode get_admin_url() . 'admin.php?page=jetpack-forms-admin', should we?
+	$handler = ( new Dashboard_View_Switch() )->get_forms_admin_url();
+
+	add_submenu_page(
+		'jetpack',
+		__( 'Jetpack Forms', 'jetpack-mu-wpcom' ),
+		__( 'Forms', 'jetpack-mu-wpcom' ),
+		'edit_pages',
+		$handler
```

Comment:
`$handler` I believe is the wrong thing here. This needs to be `jetpack-forms-admin`, not the full URL. Just the slug IIRC.